client/src/components/queueDisplay.js

```diff
@@ -1,60 +1,80 @@
 import React, { Component } from "react";
+import socketIOClient from "socket.io-client";
+import OrderNumber from "./orderNumber";
 
 //The component to represent a display of a group of orders for the persons in queue
 
 class QueueDisplay extends Component {
-    state = {};
+    state = {
+        orders: [],
+    };
+
+    // fetches order data from the backend
+    fetchOrders() {
+        fetch("/api/orders")
+            .then((res) => res.json())
+            .then((orders) =>
+                this.setState({ orders }, () =>
+                    console.log("Orders fetched...", orders)
+                )
+            );
+    }
+
+    // A listener for for updated orders from the backend
+    initiateSocket() {
+        const socket = socketIOClient("http://localhost:5000");
+        socket.on("orders-updated", (res) => {
+            this.setState({ orders: res });
+        });
+    }
+    //Method runs immediatly after render() and we make a fetch-call to the backend for the info about which orders
+    //we currently have available in the kitchen.
+    componentDidMount() {
+        this.fetchOrders = this.fetchOrders.bind(this);
+        this.fetchOrders();
+        this.initiateSocket();
+    }
 
     render() {
+        const readyOrders = {
+            orders: [...this.state.orders.filter((o) => o.status === "Redo")],
+        };
+        const unfinishedOrders = {
+            orders: [
+                ...this.state.orders.filter((o) => o.status === "Tillagas"),
+            ],
+        };
         return (
             <div className="container">
                 <div className="row">
                     <div className="col-6 border-right">
                         <h3 className="text-center">
                             <u>Tillagas</u>
                         </h3>
-                        <div className="d-flex justify-content-between">
-                            <div className="card">
-                                <div className="card-body d-flex flex-column">
-                                    <h5 className="card-title text-center">
-                                        Ordernummer: 1
-                                    </h5>
-                                </div>
-                            </div>
-                            <div className="card">
-                                <div className="card-body d-flex flex-column">
-                                    <h5 className="card-title text-center">
-                                        Ordernummer: 2
-                                    </h5>
+                        <div className="row row-cols-2 justify-content-center">
```

Comment:
```suggestion
                        <div className="row row-cols-2">
```
Tror det där ska räcka för att de ska lägga sig "korrekt" automatiskt